src/app/routes/api/v3/admin/admin.routes.ts

```diff
@@ -10,6 +10,6 @@ AdminRouter.use('/forms', AdminFormsRouter)
 
 // This endpoint doesn't reaaaallly need to be a verified admin to be used
 AdminRouter.get(
-  '/environment/:env(react|angular)',
+  '/environment/:ui(react|angular)',
```

Comment:
this was a bug yeah?

Reply:
Oopsie my bad 😰 . Thanks for that @wanlingt 🙏